ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx

```diff
@@ -207,7 +204,7 @@ function RequestCVEsTable({
                                                 />
                                             </Td>
                                             <Td dataLabel="CVSS">
-                                                <CvssTd
+                                                <CvssFormatted
```

Comment:
Nice. Excluding the `_Td` portion makes it more reusable in sections of the UI that aren't specifically related to the table.